crates/egui/src/response.rs

```diff
@@ -133,6 +67,71 @@ pub struct Response {
     /// for improved layouting.
     /// See for instance [`egui_flex`](https://github.com/lucasmerlin/hello_egui/tree/main/crates/egui_flex).
     pub intrinsic_size: Option<Vec2>,
+
+    #[doc(hidden)]
+    pub(crate) flags: Flags,
```

Comment:
I am slightly confused by the `// IN` and `// OUT` comments (still unsure what they mean), and also that all the fields are `pub`. I made this one `pub(crate)` as I really don't see why users would need to muck with it. I would even have made it so that even `Context` can't access the field directly, and has to go through accessor methods instead of messing with `flags.contains()`, `flags.set()`, etc. But I did not want to go overboard with my 1st attempt of hacking on `egui` 🙂

Reply:
Interesting. So any internal refactoring like this is accepted to be potentially breaking? Wondering if this is sustainable in the long run (if `egui` ever makes to to 1.0 😉). Also, as a new user, I found that the extra API surface doesn't make it easier to learn what I am supposed to touch, and what not. `#[doc(hidden)]` helps somewhat.